src/jdk.accessibility/windows/native/libjavaaccessbridge/AccessBridgeJavaEntryPoints.cpp

```diff
@@ -1735,11 +1743,10 @@ AccessBridgeJavaEntryPoints::getAccessibleContextInfo(jobject accessibleContext,
             EXCEPTION_CHECK("Getting AccessibleName - call to GetStringChars()", FALSE);
             wcsncpy(info->name, stringBytes, (sizeof(info->name) / sizeof(wchar_t)));
             length = jniEnv->GetStringLength(js);
+            EXCEPTION_CHECK_WITH_RELEASE("Getting AccessibleName - call to GetStringLength()", FALSE, js, stringBytes);
+            EXCEPTION_CHECK("Getting AccessibleName - call to ReleaseStringChars()", FALSE);
```

Comment:
So this is for the call to ReleaseStringChars() when there is no exception doing the GetStringLength
Since all the calls to ReleaseStringChars are now moved inside the WITH_RELEASE macro, it would seem more
logical to me to move this 2nd macro call inside the definition of the new macro (option 1)
Either that OR  (option 1) do not move the "no previous exception" case inside the new macro at all
In other words, this CHECK should be placed next to where I can see the call that requires it.
Now, you'll probably tell me it isn't needed in the cases where the method immediately returns, so if that causes
a problem then option 2 seems like it would be better.